earth_enterprise/rpms/build.gradle

```diff
@@ -541,16 +618,12 @@ task openGeeServerRpm(type: GeeRpm, dependsOn: openGeeServerInitScripts) {
     requires('opengee-postgis', '2.3.9', GREATER | EQUAL)
     conflicts('opengee-postgis', '2.0', LESS )
     conflicts('opengee-postgis', '2.4', GREATER | EQUAL)
-    requires('python-unittest2')
-    // The WMS Service requires PIL, which is not automatically detected:
-    requires('python-imaging')
-    requires('python-lxml')
 
     requiresCommands(
         packageSharedCommands +
         [
             // Install script commands:
-            'cat', 'chmod', 'chown', 'cut', 'find', 'getent', 'mkdir', 'python',
+            'cat', 'chmod', 'chown', 'cut', 'find', 'getent', 'mkdir', 'python2.7',
```

Comment:
Should we add gee-python2 and gee-python3?

Reply:
For now I don't think so. Our python rpm is more or less a 1-1 replacement of the system python packages that aren't available. Requiring just python (until we totally move to python 3) should be sufficient, and should cover both our python installs and the system ones if they user gets them.